snuba/cli/consumer.py

```diff
@@ -42,15 +52,6 @@
     multiple=True,
     help="Kafka bootstrap server to use.",
 )
-@click.option(
-    "--storage",
```

Comment:
This is going to break all current Snuba consumers in dev as well as prod since they are all passing `--storage` right now. Can we put a migration plan in place?

Reply:
Oh oops...I can do that. Maybe we can sync offline about this as well

Reply:
just in case it's useful, a click option can have multiple aliases: `@click.option("--storage", "--storages")`

Reply:
Looking at this again, I think it should probably stay singular anyway.

- It is also `--storage` on the multistorage consumer as well so making it singular would align to that
- `snuba consumer --storage errors --storage errors2` reads a lot better than `snuba consmer --storages errors --storages errors2`